icubam/backoffice/templates/list.html

```diff
@@ -16,27 +16,26 @@
 <section class="content">
   <div class="container-fluid">
 
-  <section class="content-header">
-   <div class="container-fluid">
-     <div class="row mb-2">
-       <div class="col-sm-6">
-         <h1>{{objtype}}</h1>
-       </div>
-     </div>
-   </div><!-- /.container-fluid -->
-  </section>
+    <section class="content-header">
+      <div class="container-fluid">
+        <div class="row mb-2">
+          <div class="col-sm-6">
+            <h1>{{_(objtype)}}</h1>
```

Comment:
Are we sure to have sufficient translations for all objtypes?

Reply:
If not, the strings without translations are kept unchanged.